Bio/PDB/Polypeptide.py

```diff
@@ -452,35 +452,3 @@ def _test_dist(self, c, n):
         else:
             return 0
 
-
```

Comment:
You've left a blank line at the end of the file, which is why the TravisCI style check is failing:


```
$ flake8 Bio/
Bio/PDB/Polypeptide.py:454:1: W391 blank line at end of file
```